license.rst

```diff
@@ -0,0 +1,72 @@
+==========================
+END USER LICENSE AGREEMENT
+==========================
+
+for Anaconda, Miniconda, and Conda
```

Comment:
I need to redo that page anyway, thanks for pointing that out.

Kerry

On Tue, May 19, 2015 at 1:49 PM, Aaron Meurer <notifications@github.com>
wrote:

> In license.rst
> <https://github.com/conda/conda-docs/pull/55#discussion_r30632892>:
>
> > @@ -0,0 +1,72 @@
> > +==========================
> > +END USER LICENSE AGREEMENT
> > +==========================
> > +
> > +for Anaconda, Miniconda, and Conda
>
> I'm not sure what you mean by this.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/conda/conda-docs/pull/55/files#r30632892>.
>